airssample_fr.xml

```diff
@@ -17,24 +17,24 @@
             <Confidential>false</Confidential>
         </AKA>
         <AKA>
-            <Name>FORMER_ORG 1 [French]</Name>
+            <Name>ANCIENNE_ORG 1 [French]</Name>
             <Confidential>false</Confidential>
-            <Description>Former Name : 2001</Description>
+            <Description>Nom antérieur : 2001</Description>
         </AKA>
         <AKA>
-            <Name>FORMER_ORG 2 [French]</Name>
+            <Name>ANCIENNE_ORG 2 [French]</Name>
```

Comment:
The blocked caps represent CIOC field names and should not be translated for clarity (FORMER_ORG, LEGAL_ORG, etc.). That's why we are using [French] to provide some distinction from the English version of the file.